cmd/diff.go

```diff
@@ -158,7 +158,11 @@ func diffFile(image1, image2 *pkgutil.Image) error {
 	if err != nil {
 		return err
 	}
-	util.TemplateOutput(diff, "FilenameDiff")
+	writer, err := getWriter(outputFile)
+	if err != nil {
+		return err
+	}
+	util.TemplateOutput(writer, diff, "FilenameDiff")
```

Comment:
I know you didn't make this change, but for some reason we're ignoring the error here. could you check (and wrap) the error from `TemplateOutput()`?

Reply:
aye!

Reply:
TemplateOutput returns an error, so we can check, log it, and return it if this is the case.
```golang
func diffFile(image1, image2 *pkgutil.Image) error {
	diff, err := util.DiffFile(image1, image2, filename)
	if err != nil {
		return err
	}
	writer, err := getWriter(outputFile)
	if err != nil {
		return err
	}
	util.TemplateOutput(writer, diff, "FilenameDiff")
	if err != nil {
		logrus.Error(err)
		return err
	}
	return nil
}
```